docs/specs/tripgo.swagger.yaml

```
@@ -3285,6 +3285,17 @@ definitions:
               wheelchairAccessible:
                 type: boolean
                 description: Is this stop wheelchair accessible? Missing when unknown.
+              boardingRules:
+                type: string
+                description: This appears when this stop is a "Requested Stop". The specific string defines the requested method where the `REQUEST_REQUIRED` value is the value whn the method is unknown.
```

I'd rephrase this like this:

```suggestion
                description: This appears when this stop is a "requested stop", i.e., the service won't stop here unless specifically requested by a passenger. The `REQUEST_REQUIRED` value will be used if the specific way of requesting is unknown (i.e., not provided in the source data), and one of the other values is used when the method is known...
```

@fersagui That makes me wonder: What if you can use multiple methods such as phone call or mobile app?

---

From the data we are consuming, there is always only one per stop. Maybe there is a priority between them. The other option (if we find a case) is to use a list.